cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,73 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
+    cy.viewport(1920, 1080);
   });
 
-  it('', () => {
+  it('Automation registration form', () => {
+    const firstName = 'Tanjiro';
```

Comment:
Hardcoded test data should be avoided. Consider extracting these variables into a fixture file or generating them dynamically during each test run.